.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Have you seen this [https://raw.githubusercontent.com/pypi/warehouse/main/.github/ISSUE_TEMPLATE/bug-report.md](https://raw.githubusercontent.com/pypi/warehouse/main/.github/ISSUE_TEMPLATE/bug-report.md)?